packages/central-server/src/createApp.js

```diff
@@ -1,24 +1,52 @@
 /**
- * Tupaia MediTrak
- * Copyright (c) 2017 Beyond Essential Systems Pty Ltd
+ * Tupaia
+ * Copyright (c) 2017 - 2024 Beyond Essential Systems Pty Ltd
  */
 
 import express from 'express';
 import cors from 'cors';
 import bodyParser from 'body-parser';
 import errorHandler from 'api-error-handler';
 import morgan from 'morgan';
+import publicIp from 'public-ip';
 import { Authenticator } from '@tupaia/auth';
 import { buildBasicBearerAuthMiddleware } from '@tupaia/server-boilerplate';
 import { handleError } from './apiV2/middleware';
 import { apiV2 } from './apiV2';
 
+const TRUSTED_PROXIES_INTERVAL = 60000; // 1 minute
```

Comment:
I just took a guess at this value. On the one hand I don't want to add any load to the server and on the other if the IP changes, we want to minimise downtime. Also the publicIp fetch timeout is set to 5 seconds by default so we need to be safely above that.

Reply:
I think this is fine. Tbh I don't know exactly how often the elastic IP changes, but back when we were having the bug it seemed to change maybe once every couple days. I guess there being a 1 minute window of possible errors once every couple days isn't the best, but perhaps we can see how this goes and if we start seeing errors often enough we make the interval more regular :+1: